cinema/tests/test_movie_api.py

```diff
@@ -27,9 +28,7 @@
 
 
 def sample_genre(**params):
-    defaults = {
-        "name": "Drama",
-    }
+    defaults = {"name": "Drama",}
```

Comment:
The `sample_genre` function correctly creates a genre with default parameters. Ensure that the function is used consistently across test cases to maintain uniformity.